at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
(Add a comment above it to state why the ast_verbose is so useful there, next to the ast_debug.)